habitat-lab/habitat/tasks/rearrange/multi_task/pddl_domain.py

```diff
@@ -446,9 +495,43 @@ def get_possible_actions(
                     matching_actions.append(new_action)
         return matching_actions
 
+    def get_ordered_actions(self) -> List[PddlAction]:
+        """
+        Gets an ordered list of PDDL actions.
```

Comment:
which actions? I assume the ones that are available to the agent as opposed to the ones that have been used so far, etc... 
"an ordered list of PDDL actions" is not descriptive